ownCloud/Settings/Passcode/PasscodeViewController.swift

```diff
@@ -0,0 +1,192 @@
+//
+//  PasscodeViewController.swift
+//  ownCloud
+//
+//  Created by Javier Gonzalez on 03/05/2018.
+//  Copyright © 2018 ownCloud GmbH. All rights reserved.
+//
+
+/*
+ * Copyright (C) 2018, ownCloud GmbH.
+ *
+ * This code is covered by the GNU Public License Version 3.
+ *
+ * For distribution utilizing Apple mechanisms please see https://owncloud.org/contribute/iOS-license-exception/
+ * You should have received a copy of this license along with this program. If not, see <http://www.gnu.org/licenses/gpl-3.0.en.html>.
+ *
+ */
+
+import UIKit
+
+typealias CompletionHandler = (() -> Void)
+
+class PasscodeViewController: UIViewController, Themeable {
+
+    // MARK: - Overlay
+    var hiddenOverlay: Bool?
+
+    // MARK: - Overlay view
+    @IBOutlet var overlayView: UIView!
+    @IBOutlet var logoTVGView : VectorImageView!
+
+    // MARK: - Messages and input text
+    @IBOutlet weak var messageLabel: UILabel?
+    @IBOutlet weak var errorMessageLabel: UILabel?
+    @IBOutlet weak var passcodeValueTextField: UITextField?
+    @IBOutlet weak var timeTryAgainMessageLabel: UILabel?
+
+    // MARK: - Buttons
+    @IBOutlet var numberButtons: [ThemeButton]?
+
+    @IBOutlet weak var number0Button: ThemeButton?
```

Comment:
You don't need all these buttons if you perform the `outlet collection`

Reply:
Deleted